app/views/bus_stops/_form.haml

```diff
@@ -48,4 +48,7 @@
         %td= f.check_box :completed, class: 'triple-zoom'
     %tr
       %td= submit_tag 'Save stop', class: 'btn btn-primary btn-lg btn-block submit'
+      - unless session[:route_id].nil?
```

Comment:
Alternatively, `-if session[:route_id]`, right? Curious why this was chosen instead.